app/Http/Controllers/TaskController.php

```diff
@@ -3,49 +3,66 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
-use App\Models\Task;
 use Illuminate\Support\Facades\Validator;
+use App\Models\Task;
 
-class TaskController extends Controller
+class taskController extends Controller
 {
     public function index(){
-        $tasks = Task::all(); 
-        return view('tasks', compact('tasks'));
-    }
-
-    public function create(){
-        $tasks = Task::orderBy('created_at', 'asc')->get();
- 
-        return view('task', [
-            'tasks' => $tasks
-        ]);
+        $tasks = Task::all();
+        return view('task', compact('tasks'));
     }
     
     public function store(Request $req){
+        $arr = [
+            'name' => 'required',
+            'time' => 'required',
+            'date' => 'required'
+        ];
 
-        $validator = Validator::make($req->all(), [
-        'name' => 'required | min:5 | max:20',
-        ]);
-    
-        if ($validator->fails()) {
-            return redirect('/')
-                ->withInput()
-                ->withErrors($validator);
+        $valid = Validator::make($req->all(), $arr);
+        if ($valid -> fails()) {
+            return 'Please Fill All Fields!';
         }
+        else{
 
-        $task = new Task;
-        $task->name = $req->name;
-        $task->save();
-    
-        return redirect('/');
- 
+            $drt = date('A', strtotime($req->time));
+            // return $drt;
+
+            $data = Task::where([
+                ['time', 'like', '%'. $drt .'%'],
+                ['date', '=', $req->date]
+            ])->get();
+
+            if ($data->count() > 0) {
+                return 'Time Already Exists!';
+            }
+            else{
+                
+                $h = date('H', strtotime($req->time));
+                if ($h > 12) {
+                    $h = $h - 12;
+                    $time = date($h .':i A', strtotime($req->time));
+                }
+                else{
+                    $time = date('H:i A', strtotime($req->time));
+                }
+            }
```

Comment:
you can use unique validation instead of writing this long validation conditions

Reply:
solved.